raiden/ui/cli.py

```diff
@@ -306,4 +306,4 @@ def run(ctx, **kwargs):
         gevent.signal(signal.SIGINT, event.set)
         event.wait()
 
-        app_.stop()
+        app_.stop(graceful=True)
```

Comment:
From my point of view the graceful shutdown behavior should be requested explicitly, when plugging the `App` into an actual application; the default being `False` is correct. 

For configuration switches: Unless the creator of the `raiden` application (us) can provide a secure way to handle shutdown without risk of losing tokens when omitting the cleanup ( #182 #293 #541) , we shouldn't put the switch in the user's authority. User's that don't want to wait can still force quit the application.

This may seem to contradict what I wrote above, but I think it doesn't :)

1. is dead code from what I can tell. Will delete it.
2. can be made more explicit, but the default behavior is `graceful=False` which is what we want here. I'll add the explicit switch for the default behavior.
3. Was already in ths PR :) https://github.com/konradkonrad/raiden/blob/048135e3625c60b33e142ad140196c22f64ca719/raiden/tests/fixtures/raiden_network.py#L20

as for a fixture: unless testing the graceful shutdown explicitly I don't see why we want tests to wait for cleanup of throwaway blockchain state.